internal/pkg/cli/storage_init.go

```diff
@@ -21,30 +22,37 @@ import (
 )
 
 const (
-	dynamoDBStorageType = "DynamoDB"
-	s3StorageType       = "S3"
-)
-
-const (
-	s3BucketFriendlyText      = "S3 Bucket"
-	dynamoDBTableFriendlyText = "DynamoDB Table"
-)
-
-const (
-	ddbKeyString = "key"
+	dynamoDBStorageType     = "DynamoDB"
+	s3StorageType           = "S3"
+	rdsStorageType          = "RDS"
 )
 
 var storageTypes = []string{
 	dynamoDBStorageType,
 	s3StorageType,
+	//rdsStorageType, // Hide RDS option for now.
+}
+
+var storageTypeHints = map[string]string {
+	dynamoDBStorageType: "",
+	s3StorageType: 		 "",
+	rdsStorageType: 	 "Aurora Serverless",
```

Comment:
What do you think of having this 🤔 
```
  > Aurora Serverless       (SQL) 
    DynamoDB                (NoSQL)
    S3                      (Objects)
```

Reply:
That looks good to me..! Do we want the RDS keyword anywhere though? Do you think it'd be helpful to write `RDS Aurora Serverless` instead of `Aurora Serverless`?

Reply:
I think we can do the following:
- Accepts -t "Aurora" as flag input (as opposed to "Aurora Serverless" which is too long and contains space)
- When prompting for storage type, display "Aurora Serverless     (SQL)"
- These two inputs should map to "Aurora" which is how internally we recognize the storage type.

Reply:
> I think we can do the following:
> 
>     * Accepts -t "Aurora" as flag input (as opposed to "Aurora Serverless" which is too long and contains space)
> 
>     * When prompting for storage type, display "Aurora Serverless     (SQL)"
> 
>     * These two inputs should map to "Aurora" which is how internally we recognize the storage type.

Sounds good to me :shipit: !